core/modules/views_ui/views_ui.admin.inc

```diff
@@ -2290,12 +2290,29 @@ function views_ui_edit_form_get_bucket($type, $view, $display) {
 
 /**
  * Regenerate the current tab for AJAX updates.
+ *
+ * @param view $view
+ *   The view object to work on.
+ * @param array $output
+ *   The output array.
+ * @param string $display_id
+ *   The ID of the display in the $view that is being viewed.
  */
-function views_ui_regenerate_tab(&$view, &$output, $display_id) {
+function views_ui_regenerate_tab(&$view, array &$output, $display_id) {
```

Comment:
This code hinting will cause non-array parameters to throw a fatal error (and would make the loose checking below never executed).

Reply:
removed.